core/templates/search/results_pagination.html

```diff
@@ -1,44 +1,48 @@
 {% if page.has_previous or page.has_next %}
-  <nav aria-label="Pages" class="pull-right">
+  <nav aria-label="Pages" class="float-end">
     <ul class="pagination pagination-sm">
       {% if page.has_previous %}
-        <li>
-          <a href="{{previous_url}}" aria-label="Previous Page">
+        <li class="page-item">
+          <a  class="page-link" href="{{previous_url}}" aria-label="Previous Page">
             <span aria-hidden="true">&laquo;</span>
           </a>
         </li>
       {% else %}
-        <li class="disabled">
-          <span aria-hidden="true">&laquo;</span>
+        <li class="page-item disabled">
+          <a  class="page-link" href="#" aria-label="Previous Page" aria-disabled="true">
+            <span aria-hidden="true">&laquo;</span>
+          </a>
         </li>
       {% endif %}
 
       {% for page_number in page_range_short %}
         {% ifequal page_number "..." %}
-          <li class="disabled">
-            <a>…</a>
+          <li class="page-item disabled">
+            <a class="page-link" aria-label="Middle of Page Range" href="#"> …</a>
```

Comment:
Is this actually the bootstrap way to do this? I thought one could apply the page-link style to a span to avoid having a real link disabled by javascript

Reply:
Here's the page in the Bootstrap docs about this: https://getbootstrap.com/docs/5.0/components/pagination/ I'll take a closer look before we meet

Reply:
Wrong link - apologies. Here's the page from the latest version: https://getbootstrap.com/docs/5.3/components/pagination/

Reply:
I think you're right - they have a couple examples and I think I followed the wrong one. Should it look like this instead?

```
    <li class="page-item disabled">
      <span class="page-link" aria-label="Middle of Page Range">Previous</span>
    </li>
```

Reply:
Making a note for myself that these are the files I need to change:
- [ ] issue_frontpages_ctrl.html
- [ ] issue_pages_ctrl.html
- [ ] institutions.html
- [ ] results_pagination.html

Reply:
Yes, I think the span approach is best so if javascript is off it doesn't show the user a link. We can confirm at standup!